tests/chainsaw-multi-ns/README.md

```diff
@@ -0,0 +1,40 @@
+# Multi-Namespace E2E Tests
+
+This directory contains Chainsaw tests for validating multi-namespace operator deployments.
+
+## Prerequisites
+
+These tests assume the cluster has been set up using `make deploy_multi_namespace`, which deploys:
+
+1. **CRDs** installed once in `kube-system` via `ngrok-operator-crds` helm chart
+2. **Operator A** (`ngrok-operator-a`) in `namespace-a`:
+   - Watches only `namespace-a`
+   - Uses ingress class `ngrok-a`
+   - Controller name: `k8s.ngrok.com/ingress-controller-a`
+3. **Operator B** (`ngrok-operator-b`) in `namespace-b`:
+   - Watches only `namespace-b`
```

Comment:
It might be simple to write a chainsaw test for these by making a set of ingress objects in a combination of namespaces + ingressClasses and asserting the cloud endpoint CRDs are created or checking the ingress' statuses